.github/workflows/checkstyle.yml

```diff
@@ -0,0 +1,17 @@
+name: Checkstyle
```

Comment:
Replace it by "Java Checkstyle" please

Reply:
Thanks, I changed it and also renamed the file to `java-checkstyle.yml`.